imagetest/test_suites/network/alias_ip_test.go

```diff
@@ -16,16 +16,28 @@ import (
 )
 
 const (
-	markerFile = "/boot-marker"
+	markerFile = "/var/boot-marker"
 )
 
 func TestAliases(t *testing.T) {
+	ctx := utils.Context(t)
+	if image, err := utils.GetMetadata(ctx, "instance", "image"); err != nil {
+		t.Fatalf("could not determine image: %v", err)
+	} else if strings.Contains(image, "sles-15") || strings.Contains(image, "opensuse-leap") || strings.Contains(image, "ubuntu-1604") {
```

Comment:
I wonder if it's better to just avoid running these tests in the first place (leaving them out of the list of tests in the `setup` step) instead of skipping in the actual test.

Reply:
Yeah that's probably better. Done.